tests/unit/views/pacbio/PacbioRunIndex.spec.js

```diff
@@ -1,17 +1,16 @@
-import PacbioRuns from '@/views/pacbio/PacbioRunIndex'
-import { mount, store, Data, flushPromises } from '@support/testHelper'
+import PacbioRunIndex from '@/views/pacbio/PacbioRunIndex'
 import Response from '@/api/Response'
+import { mount, store, Data, flushPromises } from '@support/testHelper'
 
-describe('Runs.vue', () => {
-  let wrapper, runs, mockRuns
+describe('PacbioRunIndex.vue', () => {
+  let wrapper, pacbioRunIndex, mockRuns, mockVersions
 
   beforeEach(async () => {
     mockRuns = new Response(Data.PacbioRuns).deserialize.runs
-    const get = vi.spyOn(store.state.api.traction.pacbio.runs, 'get')
-    get.mockReturnValue(Data.PacbioRuns)
+    vi.spyOn(store.state.api.traction.pacbio.runs, 'get').mockResolvedValue(Data.PacbioRuns)
```

Comment:
Does the SMRTLink Version call need to be mocked here as well or is it not called?